src/app/header/header.component.html

```diff
@@ -38,21 +38,35 @@
           </span><span></span>
           Product Management
         </a>
-        <ng-container *ngIf="this.helper.localStorageItem('email') == null; else nobody">
-          <a class="navbar-item" routerLink="/login" routerLinkActive="is-active">
-            <span class="icon">
-              <i class="fas fa-user"></i>
-            </span><span></span>
-            Login
-          </a>
-        </ng-container>
-        <ng-template #nobody>
-          <a class="navbar-item" ng-click="userService.logout()"> <span></span> Logout</a>
-          <span class="icon">
-            <i class="fas fa-user"></i>
-          </span>
-        </ng-template>
-      </div>
+
+          <ng-container *ngIf="this.helper.localStorageItem('email') == null">
```

Comment:
can we just do an if else here?

Reply:
I do not know, but i know for a fact this works